### DISCUSSION TITLE
Update on  Bootstrap 5 support

### DISCUSSION THREAD

**Discussion by ganrath** [snapappointments/bootstrap-select#2762] [General]
Hi @caseyjhol , 

We are trying to  upgrade Bootstrap version to  5.x in our product due to security concerns. However bootstrap Select  is not working with latest release **[v1.14.0] beta3**.  Still they are open issue. can we expect a bugfree version soon ? if so, can you provide a date? that would be helpful

**Comment by cwolcott** [comment]
I am using it successfully with Bootstrap 5.1.3, but noticed that the items are duplicated when I try to disable the element, caused by refresh.
```                 
$('#selectorResearchType').prop('disabled', false);            
$('#selectorResearchType').selectpicker('refresh');
```

**Reply by ganrath** [reply]
@cwolcott  : Its known/open  issue only refer below link
https://github.com/snapappointments/bootstrap-select/issues/2712
